CHANGELOG.md

```diff
@@ -44,6 +44,8 @@
 
 * Major update of all documentation contents.
 * Add Sphinx roles for GitHub links ({ghcommit}`2789499b3ba66f2b00c1a0987fdaa9cdc1f5f705`).
+* Added figures to illustrate Eradiate's conventions and specified convention
+  used for spherical coordinates ({ghpr}`192`).
```

Comment:
Let's try and keep this concise:
```suggestion
* Major update of all documentation contents ({ghpr}`192` and many commits).
* Add Sphinx roles for GitHub links ({ghcommit}`2789499b3ba66f2b00c1a0987fdaa9cdc1f5f705`).
```
Docs updates are a lot of work but they have no impact on the technical side.